resources/.ktlint_editorconfig

```diff
@@ -4,3 +4,11 @@ ktlint_standard_max-line-length = 160
 ktlint_standard = enabled
 ktlint_standard_multiline-expression-wrapping = disabled
 ktlint_standard_string-template-indent = disabled
+ktlint_standard_chain-method-continuation = disabled
+ktlint_standard_class-signature = disabled
+ktlint_standard_trailing-comma-on-call-site = disabled
+ktlint_standard_function-signature = disabled
```

Comment:
🔧 As above, though in this case I think it is very clearly `ktlint_function_signature_rule_force_multiline_when_parameter_count_greater_or_equal_than`  that needs to be set (or unset).

Reply:
For all cases, the rules I disabled are the ones that ktlint reports as broken when running the linting. Is this just a more specific rule filter than the one I have disabled?

Reply:
Yeah, this more specific rule does the same thing. It's unfortunate that it doesn't give feedback on what particular rule is broken whenever we lint, but just the general type of rule :P